src/analysis/souffle/tests/arcs_fact_tests/fail_check_on_subpaths.dl

```diff
@@ -0,0 +1,34 @@
+// Original Arcs manifest file:
+// Note: Renamed the tags "trusted" and "loggable" to "userSelection" and "screenContent", as they
+// propagate like confidentiality tags.
+// #Ingress: P1
+// #FAIL: hc:P2.foo is userSelection
+//particle P1
+//  foo: writes Foo {a: Text, b: Number}
+//  claim foo.a is userSelection
+//  claim foo.b is screenContent
+//particle P2
+//  foo: reads Foo {a: Text, b: Number}
+//  check foo is userSelection
```

Comment:
This is the test case where the tags for `foo` are computed from the tags for the fields `foo.a` and `foo.b`. In our case, foo should have the confidentiality tags `{userSelection, screenContent}`. 

The check is trying to establish that `foo` *definitely* has the label `userSelection`.  Since `foo.b` does not have the `userSelection` tag, this check is supposed to fail.  Without a `definitelyHasTag` relation, we cannot validate this check, but we can verify that the correct tags are inferred.

Reply:
I see, so I'll add the checks you mentioned for the privacy aspect now and add a comment at the top saying that we need to add the integrity aspect later.

Reply:
Done.